git-branchless/src/opts.rs

```diff
@@ -19,6 +19,12 @@ impl FromStr for Revset {
     }
 }
 
+impl Default for Revset {
```

Comment:
I don't think we should set `Default` for `Revset`, as it doesn't make sense as a general-purpose default value. For example, many of the command take a revset which defaults to `stack()` instead.

Reply:
👍  Removed.